.github/workflows/push.yml

```
@@ -263,7 +271,8 @@ jobs:
       - prepare
     runs-on: arc-docker
     timeout-minutes: 45
-    if: needs.prepare.outputs.BUILD_CHUNKS
+
+    if: ${{ needs.prepare.outputs.BUILD_CHUNKS && ((needs.pre-checks.outputs.PRE_CHECK != 'push') || needs.prepare.IS_JUDICIAL_AFFECTED == 'true') }}
```

actionlint (1.7.4)

property "is_judicial_affected" is not defined in object type {outputs: {build_chunks: string; cache_key: string; deploy_feature: string; docker_tag: string; images: string; is_judicial_affected: string; last_good_build_docker_tag: string; node-modules-hash: string; node_image_version: string; test_chunks: string}; result: string}

(expression)